hopsworks-dela/src/main/java/io/hops/hopsworks/dela/hopssite/util/HopsSiteEndpoints.java

```diff
@@ -5,22 +5,22 @@
 @Stateless
 public class HopsSiteEndpoints {
```

Comment:
@o-alex Seems this class is not used anymore. Should it be removed before the PR is merged?

Reply:
removed the dead code and also moved to use the default method for creating a new agent